### PR TITLE
Rename & rebuild after migration

### PR DESCRIPTION
**Pull request by vavsab**

### CODE REVIEW

**Comment by vavsab** [comment]
@ringods Could you pls help me with the publishing secrets? Looks like they are not set for this repo

**Comment by ringods** [comment]
@vavsab publishing secrets are configured at organization level.

**Review by ringods — DISMISSED** [review]

**Comment by vavsab** [comment]
@ringods could you pls create an npm package for me?
https://github.com/pulumiverse/pulumi-zitadel/actions/runs/5115178524/jobs/9196266854?pr=1#step:14:291

And also please check that all repo-level secrets are removed. I used my personal npm secret before transfering it to you.

**Comment by vavsab** [comment]
@ringods Could you pls help me here? Looks like I cannot fix it from my side

**Comment by ringods** [comment]
@vavsab I removed the repository level NPM secret. Does that unblock you going forward?

**Review by ringods — DISMISSED** [review]

**Comment by vavsab** [comment]
@ringods please review 🙏

File: README.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ This package is available for several languages/platforms:
 To use from JavaScript or TypeScript in Node.js, install using either `npm`:
 
 ```bash
-npm install @vavsab/pulumi-zitadel
+npm install @pulumiverse/pulumi-zitadel
```
Comment (ringods) [inline README.md]: The correct NPM package name is `@pulumiverse/zitadel`.

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vavsab) [reply README.md]: Fixed

**Review by ringods — APPROVED** [review]

**Pull request merged**